Library/Homebrew/cask/cmd/doctor.rb

```diff
@@ -33,7 +33,6 @@ def run
       end
 
       def check_software_versions
-        ohai "Homebrew Cask Version", Hbc.full_version
```

Comment:
Can we change this to `HOMEBREW_VERSION`?

Reply:
Changed.

Reply:
Thanks!